qiskit/circuit/library/n_local/n_local.py

```diff
@@ -660,7 +659,7 @@ def initial_state(self) -> Any:
         return self._initial_state
```

Comment:
I can't comment there but there's one `Any` left a few lines above from here 😄

Reply:
and I found a third one. Have a look to c49ff90dd87e7e27657fbc06af09bed91419fb94 to check it makes sense